src/components/PhoneNumberInput/index.js

```diff
@@ -29,8 +29,10 @@ class PhoneNumberInput extends Component {
     this.props.actions.setMobileNumber({number, valid})
   }
 
+  handleEnterKey = (e) => { if (e.key === 'Enter') { e.preventDefault() } }
+
   render = ({i18n}) =>
-    <form>
```

Comment:
Why not just remove the `form` wrapper?

Reply:
Because we use `react-phone-number-input` and without wrapping it in a form the chrome autocomplete doesn't work. It's in their documentation.

Reply:
Ah, cool - makes sense

You should probably go for `onSubmit={e => e.preventDefault()}` instead, since it's a little less hacky (and defends in case the form gets submitted in some other way)

Reply:
You are right. I will change it now